### PR TITLE
[SMALLFIX] Replace anonymous type with lambda in alluxio.worker.AlluxioWorkerRestServiceHandler

### PR DESCRIPTION
**Pull request by WangYuxincs**
Replace anonymous type with lambda in alluxio.worker.AlluxioWorkerRestServiceHandler

### CODE REVIEW

**Comment by apc999** [comment]
@WangYuxincs can you resolve the merge conflict?

**Comment by WangYuxincs** [comment]
@apc999 The merge conflict has been resolved.

**Comment by WangYuxincs** [comment]
@apc999 Need I pull a new request?

File: core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java [hunk 1]
```diff
@@ -311,7 +311,7 @@ public Long call() throws Exception {
   @ReturnType("java.lang.Long")
   @Deprecated
   public Response getUptimeMs() {
-    return RestUtils.call(() -> mWorkerProcess.getUptimeMs());
+    return RestUtils.call(()-> mWorkerProcess.getStartTimeMs());
```
Comment (apc999) [inline core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java]: is this correct?
should still be `getUptimeMs` right?

Comment (WangYuxincs) [reply core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java]: Sorry, I made mistakes. I think I need pull a new request.

**Pull request closed without merging**